### DISCUSSION TITLE
What image generations speeds are you getting on google colab?

### DISCUSSION THREAD

**Discussion by juan9999** [TheLastBen/fast-stable-diffusion#1037] [General]
I'm on a T4 GPU and using SD 1.5 and getting the following numbers generating 512X512 images:  

<img width="304" alt="Screen Shot 2022-12-15 at 9 33 29 AM" src="https://user-images.githubusercontent.com/31065728/207905853-d980d4a8-9dd5-4fe4-a886-d10853ae182d.png">

I tried renting an A100 and got similar numbers.

The host of lukium.ai tells me he is getting around 18 it/s on test img at 512x512 on a 3090 so I was wondering why google colab was so slow. Is there anything to do to speed it up?

Are other environments faster (runpod, ...)?

HELP!

**Comment by TheLastBen** [comment]
with the T4 you get 6.2 it/s for 512 images with euler sampler, try the T4

**Reply by juan9999** [reply]
This was on the T4- tried the A100 and got similar results too :/ 

Is this with colab pro? (That's what I'm using) or a dedicated server?

**Reply by TheLastBen** [reply]
make sure you are using the latest colab https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb, because with the standard GPU you should get 6 it/s and with the A100 at least 13 it/s

**Reply by juan9999** [reply]
I did use the latest :/

I'll try again later today.

**Comment by juan9999** [comment]
Is there something I can add to the notebook to monitor the performance or memory of the GPU while it's generating? 

I run a !nvidia-smi at the star. if yes, that what and where.
